Tests/AllToolsUtilities.Tests.ps1

```diff
@@ -63,6 +63,9 @@ Describe "AllToolsUtilities.psm1" {
             Mock Install-ContainerToolConsent -ModuleName 'AllToolsUtilities' -MockWith { return $true }
             Mock Install-ContainerTools -ModuleName 'AllToolsUtilities'
             Mock Test-EmptyDirectory  -ModuleName 'BuildkitTools' -MockWith { return $true }
```

Comment:
Please change `BuildkitTools` to `AllToolsUtilities`. This method `Test-EmptyDirectory` may not even be called in this script. We should consider removing it altogether